daemon/mgr/container.go

```diff
@@ -589,6 +589,11 @@ func (mgr *ContainerManager) Start(ctx context.Context, id string, options *type
 		return err
 	}
 
+	// check if container's status is paused
+	if c.IsPaused() {
+		return fmt.Errorf("Cannot start a paused container, try unpause instead")
```

Comment:
in golang, we don't start with capital letter for error.

```
Error strings should not start with a capital letter because they'll often be prefixed before printing
```

https://github.com/golang/go/wiki/Errors